### PR TITLE
Update readme with new props docs

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Comment by qur786** [comment]
image

File: README.md [hunk 1]
```diff
@@ -79,14 +79,17 @@ interface CalendarHeatmapDatum {
...
+| Property | Type                                                                                 | Usage                                                                  |   Default   | Required |
+|:---------|:-------------------------------------------------------------------------------------|:-----------------------------------------------------------------------|:-----------:|:--------:|
+| data     | `CalendarHeatmapDatum[]`                                                             | Time series data spanning over 1 year or more years                    |    none     |   yes    |
+| color    | color hex code, valid css color name or color scheme names (`'spectral'` or `'hsl'`) | Theme color for the visual elements                                    | `'#ff4500'` |    no    |
+| overview | `'global' \| 'year' \| 'month' \| 'week' \| 'day'`                                   | Initial overview for the map                                           |  `'year'`   |    no    |
+| response | `'hide' \| 'rotate' \| 'offset'`                                                     | Responsiveness strategy for handling overlapping axis labels           |  `'hide'`   |    no    |
```
Comment (maneetgoyal) [inline README.md]: Looks incorrecr:

```
export interface DrilldownCalendarProps
  extends Omit<HTMLProps<HTMLDivElement>, 'data'> {
  data?: CalendarHeatmapDatum[];
  color?: string;
  overview?: DrilldownCalendarOverview;
  response?: Response;
  onTooltip?: (datum: { value: unknown }) => void;
  onHideTooltip?: () => void;
  fetchGlobalData?: () => Promise<CalendarHeatmapDatum[]>; // For fetching global data for 'global', 'year', and 'month' overview heatmap.
  fetchDayData?: DayOverviewHeatmapProps['fetchDayData'];
}
```

**Review by maneetgoyal — CHANGES_REQUESTED** [review]

**Comment by qur786** [comment]
image

File: README.md [hunk 2]
```diff
@@ -79,14 +79,16 @@ interface CalendarHeatmapDatum {
...
+| color         | color hex code, valid css color name or color scheme names (`'spectral'` or `'hsl'`) | Theme color for the visual elements                                    | `'#ff4500'` |    no    |
+| overview      | `'global' \| 'year' \| 'month' \| 'week' \| 'day'`                                   | Initial overview for the map                                           |  `'year'`   |    no    |
+| response      | `'hide' \| 'rotate' \| 'offset'`                                                     | Responsiveness strategy for handling overlapping axis labels           |  `'hide'`   |    no    |
+| onTooltip     | `(datum: { value: unknown }) => void;`                                               | onTooltip function is fired on "mouseover" over a visual element       |    none     |    no    |
+| onHideTooltip | `() => void;`                                                                        | onHideTooltip function is fired on "mouseout" over a visual element    |    none     |    no    |
+| fetchGlobalData | `() => Promise<CalendarHeatmapDatum[]>;` | To fetch data for 'global', 'year' or 'month' overview heatmap from a rest api | none | no
```
Comment (maneetgoyal) [inline README.md]:
```suggestion
| fetchGlobalData | `() => Promise<CalendarHeatmapDatum[]>;` | To fetch data for 'global', 'year' or 'month' overview heatmap from a REST API | none | no
```

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**